data/sles4sap/qe_sap_deployment/mr_test_gcp.yaml

```diff
@@ -1,29 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
 provider: 'gcp'
 apiver: 3
 terraform:
   variables:
-    project: "ei-sle-qa-sap-8469"
-    region: "%PUBLIC_CLOUD_REGION%"
+    project: 'ei-sle-qa-sap-8469'
+    region: '%PUBLIC_CLOUD_REGION%'
     deployment_name: '%QESAP_DEPLOYMENT_NAME%'
-    admin_user: "cloudadmin"
-    os_image: "%SLE_IMAGE%"
-    private_key: "~/.ssh/id_rsa"
-    public_key: "~/.ssh/id_rsa.pub"
-    gcp_credentials_file: "/root/google_credentials.json"
+    admin_user: 'cloudadmin'
+    os_image: '%QESAP_CLUSTER_OS_VER%'
```

Comment:
This will need a related MR in the job groups, and GCE tests don't have this setting defined.

Reply:
yes, also other settings has to be configured before to merge this PR. There's a Warning section in the PR description.